code/game/objects/items/devices/scanners.dm

```diff
@@ -181,15 +196,16 @@ SLIME SCANNER
 	msgs += "<span class='notice'>Analyzing Results for [H]:\nOverall Status: [status]"
 	msgs += "Key: <span class='healthscan_oxy'>Suffocation</span>/<font color='green'>Toxin</font>/<font color='#FFA500'>Burns</font>/<font color='red'>Brute</font>"
 	msgs += "Damage Specifics: <span class='healthscan_oxy'>[OX]</span> - <font color='green'>[TX]</font> - <font color='#FFA500'>[BU]</font> - <font color='red'>[BR]</font>"
+
 	if(H.timeofdeath && (H.stat == DEAD || (HAS_TRAIT(H, TRAIT_FAKEDEATH))))
 		msgs += "<span class='notice'>Time of Death: [station_time_timestamp("hh:mm:ss", H.timeofdeath)]</span>"
 		var/tdelta = round(world.time - H.timeofdeath)
 		if(H.is_revivable() && !DNR)
 			msgs += "<span class='danger'>Subject died [DisplayTimeText(tdelta)] ago, defibrillation may be possible!</span>"
 		else
-			msgs += "<font color='red'>Subject died [DisplayTimeText(tdelta)] ago.</font>"
+			msgs += "<font color='red'>Subject died [DisplayTimeText(tdelta)] ago. <b>defibrillation is not possible!</b></font>"
```

Comment:
I think you want "Defibrilation" uppercase here, since it's following a period?

Reply:
Fixed, typo.